compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -551,6 +551,7 @@ class Definitions {
     @tu lazy val Seq_drop         : Symbol = SeqClass.requiredMethod(nme.drop)
     @tu lazy val Seq_lengthCompare: Symbol = SeqClass.requiredMethod(nme.lengthCompare, List(IntType))
     @tu lazy val Seq_length       : Symbol = SeqClass.requiredMethod(nme.length)
+    @tu lazy val Seq_++           : Symbol = SeqClass.requiredMethod(nme.PLUSPLUS)
```

Comment:
`++` is defined on `scala.collection.Iterable` which is a super type of `Seq`. Maybe you should move `Seq_++` to `Iterable_++`.